listenbrainz/webserver/views/recommendations_cf_recording_api.py

```diff
@@ -0,0 +1,135 @@
+import listenbrainz.db.user as db_user
+import listenbrainz.db.recommendations_cf_recording as db_recommendations_cf_recording
+
+from listenbrainz.webserver.errors import APIBadRequest, APINotFound
+
+from flask import Blueprint, jsonify, request
+from listenbrainz.webserver.decorators import crossdomain
+from listenbrainz.webserver.rate_limiter import ratelimit
+
+recommendations_cf_recording_api_bp = Blueprint('recommendations_cf_recording_v1', __name__)
+
+
+@recommendations_cf_recording_api_bp.route("/user/<user_name>/artist")
+@crossdomain()
+@ratelimit()
+def get_recommendations(user_name):
+    """ Get recommendations for user ``user_name``.
+
+        A sample response from the endpoint may look like::
+
+        {
+          "payload": {
+
+            "last_updated": "Sat, 06 Jun 2020 08:18:50 GMT",
```

Comment:
Would prefer if we were consistent with other endpoints and kept this a unix timestamp.